### ISSUE TITLE
Using scalar parameter does not return a value

### ISSUE THREAD

**Issue by charlie662** [nuintun/node-adodb#32]
When using the posted node-adodb example database and code I'm failing to retrieve a @@Identity response when the query executes:
`connection
  .execute(
    'INSERT INTO Users(UserName, UserSex, UserAge) VALUES ("Newton", "Male", 25)',
    'SELECT @@Identity AS id'
  )
  .on('done', function(data) {
    console.log('result:', JSON.stringify(data, null, '  '));
  })
  .on('fail', function(error) {
    // TODO 逻辑处理
  });`
All that is returned in the data object is: 
`result: { "valid": true, "message": "Execute SQL: INSER INTO Users(UserName,UserSex,UserAge) VALUES(\"Newtonz", \"Male\", 25)  success!"}
`
There is no attribute relating to the @@Identity sql statement. (I wanted to use this to return the autoNumber id value when data is inserted).

**Comment by nuintun** [comment]
See [test.js line96](https://github.com/nuintun/node-adodb/blob/master/test/test.js?ts=2#L96)

**Comment by charlie662** [comment]
[test.js line96](https://github.com/nuintun/node-adodb/blob/master/test/test.js?ts=2#L96) shows that the expected response should be a `data` object with an attribute of `id` equal to 5 (or whatever the autonumber has been incremented to), and a `message` string equal to the executed sql. My response only had a data attribute confirming the success of the insert sql but with no reference to the scalar statement.
Looking through adodb.js I saw that there was both a `execute` and a `executeScalar` function. The example does not reference the latter so I thought it was overloaded or something; but when I changed the example code in README.md from `.execute` to `.executeScalar` the result was correct.
It still does not return a second `message` parameter object with `on.('done')` but the first `data` parameter object now includes a records array. Console output:
`Exec: executeScalar
Params: {
  "connection": "Provider=Microsoft.Jet.OLEDB.4.0;Data Source=node-adodb.mdb;",
  "sql": "INSERT INTO Users(UserName, UserSex, UserAge) VALUES (\"Newton\", \"Ma
le\", 26)",
  "scalar": "SELECT @@Identity AS id"
}
data: {
  "valid": true,
  "message": "Execute Scalar SQL: INSERT INTO Users(UserName, UserSex, UserAge)
VALUES (\"Newton\", \"Male\", 26) / SELECT @@Identity AS id success !",
  "records": [
    {
      "id": 13
    }
  ]
}
message: undefined`
So does this mean your README.md / README-EN.md scalar example just needs updating from referencing .execute to .executeScalar?
